### ISSUE TITLE
Documentation: Building .NET Framework ready for packing

### ISSUE THREAD

**Issue by LiamDHall** [velopack/velopack#73]
No so much of a issue, more of a wanted to let you/other know.

For .NET Framework applications you can not run

`dotnet publish -c Release --self-contained -r win-x64 -o .\publish`

instead you can run

$projectPath = "C:\Path\To\Your\Project.csproj"
$outputDirectory = ".\publish"
msbuild $projectPath `
    /p:Configuration=Release `
    /p:Platform=x64 `
    /p:OutDir=$outputDirectory `
    /t:Rebuild `
    /restore `
    /verbosity:minimal `
    /nologo
 
Making sure your publish path is at the same level as your solution (.sln file location) to then run
`vpk pack -u YourAppId -v 1.0.0 -p .\publish -e yourMainApp.exe`

If you have publish via ClickOnce before then the msbuild command above may produce a file called 
YourAppName.application

This will cause the error

`[11:56:36 FTL] Command WindowsPackCommand had an exception.
    ArgumentException: Velopack does not support building releases for ClickOnce applications. Please publish your application to a folder without ClickOnce.
      at Task<string> PreprocessPackDir(Action<int> progress, string packDir) in WindowsPackCommandRunner.cs:34
      at void MoveNext() in PackageBuilder.cs:113
      at async Task RunTask(string name, Func<Action<int>, Task> fn) in SpectreConsole.cs:130
      at void MoveNext() in PackageBuilder.cs:111
      at void MoveNext() in SpectreConsole.cs:32
      at void MoveNext() in Progress.cs:103
      at void MoveNext() in Progress.cs:138
      at async Task<T> RunAsync<T>(Func<Task<T>> func) in DefaultExclusivityMode.cs:40
      at async Task<T> StartAsync<T>(Func<ProgressContext, Task<T>> action) in Progress.cs:121
      at async Task StartAsync(Func<ProgressContext, Task> action) in Progress.cs:101
      at async Task ExecuteProgressAsync(Func<IFancyConsoleProgress, Task> action) in SpectreConsole.cs:21
      at async Task Run(T options) in PackageBuilder.cs:109
      at void MoveNext() in Program.cs:186`
      
To solve this error just delete `YourAppName.application` from the publish folder

Just wanted to let everyone know who might be having the same issue.

**Comment by caesay** [comment]
Yes, your `msbuild` command is fine. In fact, you can build your app any way you please, via F6 in visual studio for example.

Deleting `YourAppName.application` from the output directory is _not_ the correct solution. The fact that this file is being created means you have not properly removed clickonce from your project. Please edit your `.csproj` and remove all clickonce related properties, and this file will no longer be created in your build output. 

Thanks!
